src/App.js

```diff
@@ -1,25 +1,43 @@
-import React, { Component} from 'react';
-import './App.css';
-import movieData from './MovieData';
-import Movies from './Movies'
+import React, { Component } from "react";
+import "./App.css";
+import movieData from "./MovieData";
+import Movies from "./Movies";
+import MovieDetails from "./MovieDetails";
 
 class App extends Component {
-    constructor() {
-      super();
-      this.state = {
-        movies: movieData.movies
-      }
-    }
-    render() {
-      return (
-        <div>
-          <nav>
-            <h1>Rancid Tomatillos</h1>
-          </nav>
-          <Movies movies={this.state.movies} />
-        </div>
-      );
-    }
+  constructor() {
+    super();
+    this.state = {
+      movies: movieData.movies,
+      selectedMovie: 0,
+    };
+  }
+
+  selectMovie = (movie) => {
+    this.setState({ selectedMovie: movie.id });
+  };
+
+  closeDetails = () => {
+    this.setState({ selectedMovie: 0 });
+  };
+
+  render() {
+    return (
+      <>
+        <nav>
+          <h1>Rancid Tomatillos</h1>
+        </nav>
+        {this.state.selectedMovie ? (
+          <MovieDetails
+            movie={this.state.movies[0]}
```

Comment:
I think we'll eventually want to be doing a find by selected movie here but also I think we'll be refactoring this in react router before long so expect to see this line of code often over the next week I guess.